diff --git a/dist/openapi-types.d.ts b/dist/openapi-types.d.ts
index e15d592a352573aabf356a4c6fb1730173809023..4710476baddb2ce8304fa386dbc1d193b4f2bcb9 100644
--- a/dist/openapi-types.d.ts
+++ b/dist/openapi-types.d.ts
@@ -11,7 +11,7 @@ type AnyOtherAttribute = {
     /** OpenAPI extension */
     [customExtension: `x-${string}`]: any;
     /** Unknown attribute */
-    [key: string]: any;
+    // [key: string]: any;
 };
 export declare namespace OpenAPI {
     type Document<T extends AnyOtherAttribute = {}> = OpenAPIV2.Document<T> | OpenAPIV3.Document<T> | OpenAPIV3_1.Document<T> | OpenAPIV3_2.Document<T>;
